client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -461,7 +462,8 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 		<StepContainer
 			stepName="domains"
 			isWideLayout={ true }
-			hideBack={ true }
+			shouldStickyNavButtons={ isCopySiteFlow( flow ) }
```

Comment:
What other flows should we test to review these changes? The pull request description doesn't seem to indicate any.

Reply:
I don't think there are other flows that we need to test. All those changes are happening only if it's a copy site flow, otherwise, they are the same as they were before.

Reply:
If this code is used by other signup flows, it'd be good to test at least one of them to make sure we haven't accidentally broken something.